etc/perf-pr-comment.sh

```diff
@@ -4,6 +4,20 @@
 
 set -eux
 
+# Generate perf report.
 pushd ./internal/cmd/perfcomp >/dev/null || exist
-GOWORK=off go run main.go --project="mongo-go-driver" ${VERSION_ID}
+GOWORK=off go run main.go --project="mongo-go-driver" ${VERSION_ID} > ./parseperfcomp/perf-report.txt
```

Comment:
Instead of writing a text file and then reformatting it as a Markdown later, is it possible to write the final Markdown result here?